### ISSUE TITLE
Allow Item div title attribute to be configurable

### ISSUE THREAD

**Issue by rogchap** [namespace-ee/react-calendar-timeline#29]
https://github.com/namespace-ee/react-calendar-timeline/blob/master/src/lib/items/Item.jsx#L353

``` jsx
return (
      <div key={this.itemId}
           ref='item'
           className={classNames}
           title={this.itemTitle} // <--- Configurable 
```

Use case:
For my Item title I'm passing a React node and not a string, which means that the title reads `[object, Object]`. e.g.

``` jsx
const items = [
  { id:1, group:1, title:<span><Avatar /> Bruce Wayne</span> }
]
```
Everything renders correctly (which is cool), but the tooltip title is incorrect.

**Comment by xmityaz** [comment]
This issue mentioned in  #21 
Probably the best thing to do here is to create a PR with splitting the `title` into `itemTitle` and `itemContent` to be able to set tooltip title separately.

**Comment by mariusandra** [comment]
Hey, I released 0.8.3 with support for this. In the [keys hash](https://github.com/namespace-ee/react-calendar-timeline#keys) there is now a thing called "itemDivTitle". Set that to whatever you want to appear in the html title tag.
